src/App.tsx

```diff
@@ -15,71 +16,95 @@ export const goods = [
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  selectGood = (good: string) => {
+    this.setState({ selectedGood: good });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  render() {
+    const { selectedGood } = this.state;
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        {selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={this.clearSelection}
+              />
+            </h1>
+          )
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+          : (
+            <h1 className="title">
+              No goods selected
+            </h1>
+          )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map((good: string) => {
+              return (
+                <tr
+                  key={good}
+                  data-cy="Good"
+                  className={classNames(
+                    selectedGood === good && 'has-background-success-light',
+                  )}
+                >
+                  <td>
+                    {selectedGood === good
+                      ? (
+                        <button
+                          data-cy="RemoveButton"
+                          type="button"
+                          className="button is-info"
+                          onClick={this.clearSelection}
+                        >
+                          -
+                        </button>
+                      )
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+                      : (
+                        <button
+                          data-cy="AddButton"
+                          type="button"
+                          className="button"
+                          onClick={() => {
+                            this.selectGood(good);
+                          }}
```

Comment:
```suggestion
                          onClick={() => this.selectGood(good)}
```
you can leave it on the same line if it's not too long